frontend/packages/dev-console/src/components/import/import-types.ts

```diff
@@ -133,7 +133,7 @@ export interface GitData {
   dir: string;
   showGitType: boolean;
   secret: string;
-  isUrlValidated: boolean;
+  isUrlValidated?: boolean;
```

Comment:
Is this still boolean? I think it should also be
```suggestion
  isUrlValidated?: ValidatedOptions;
```